### PR TITLE
Update from upstream 20180515

### PR DESCRIPTION
**Pull request by rdickey-r7**

### CODE REVIEW

File: kong/resource_kong_consumer_credential_basic_auth.go [hunk 1]
```diff
@@ -148,7 +148,6 @@ func resourceKongBasicAuthCredentialDelete(d *schema.ResourceData, meta interfac
 	return nil
 }
 
-//TODO: pasword should be SHA1 hashed to avoid differences on refresh - https://github.com/Mashape/kong/blob/master/kong/plugins/basic-auth/crypto.lua
```
Comment (erran-r7) [inline kong/resource_kong_consumer_credential_basic_auth.go:151]: @rdickey-r7 did you include the patch to use the hash?

File: kong/resource_provider.go [hunk 2]
```diff
@@ -40,6 +40,11 @@ func Provider() terraform.ResourceProvider {
 			"kong_consumer_key_auth_credential":   resourceKongKeyAuthCredential(),
 			"kong_consumer_jwt_credential":        resourceKongJWTCredential(),
 			"kong_api_plugin_key_auth":            resourceKongKeyAuthPlugin(),
+			"kong_consumer_acl_group":             resourceKongConsumerACLGroup(),
```
Comment (erran-r7) [inline kong/resource_provider.go:43]: I added `resourceKongConsumerACL` before the acl group got landed. Double check if they do the same thing if so let's mark it as deprecated if possible.

**Comment by erran-r7** [comment]
Just realised I'd never merged this. You may need to rebase after rapid7/terraform-provider-kong#9.

**Comment by erran** [comment]
Closing due to issue age. We're long due a pull from the upstream repository so will look at that shortly if the official kong provider is not in a state where Rapid7 consumers can use it.

**Pull request closed without merging**